cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/Differ.java

```diff
@@ -87,16 +90,22 @@ public DiffNode createDiffTree(Bean liveConfig, Bean desiredConfig) {
     private DiffNode doCreateDiffTree(Bean liveConfig, Bean desiredConfig) {
         Diff diff = JAVERS.compare(liveConfig, desiredConfig);
 
+        log.verbose("Parsing JaVers change objects to custom change objects");
         // parse the change objects created by the JaVers diff to custom change objects
         List<CfChange> cfChanges = diff.getChanges()
                 .stream()
+                .peek(change -> log.verbose("JaVers found a change:", change))
                 .map(this.changeParser::parse)
                 .flatMap(Collection::stream)
                 // apply all custom set filters
                 .filter(this::applyFilterCriterion)
                 .collect(Collectors.toList());
+        log.verbose("Parsing JaVers change objects to custom change objects completed");
 
-        return DiffTreeCreator.createFrom(cfChanges);
+        log.verbose("Creating diff tree");
+        DiffNode root = DiffTreeCreator.createFrom(cfChanges);
+        log.verbose("Creating diff tree completed");
```

Comment:
If you make the above `log.verbose("Creating diff tree");` call `info`, then this logging should be `info` too in my opinion

Reply:
This would not be consistent with the rest of the logging in other classes.